src/Drivers/WindowsDecompiler/WindowsDecompiler.csproj

```diff
@@ -216,7 +216,6 @@
     <Environments Include="$(SolutionDir)Environments\ZX81\$(OutDir)Reko.Environments.ZX81.*" />
     <Libraries Include="$(SolutionDir)Libraries\Libc\$(OutDir)Reko.Libraries.Libc.*" />
     <Libraries Include="$(SolutionDir)Libraries\Python\$(OutDir)Reko.Libraries.Python.*" />
-    <ExternalComponents Include="$(SolutionDir)..\external\Capstone.NET\*.dll" />
```

Comment:
The missing <ExternalComponents /> element here will cause failures when using the Windows Reko GUI on ARM binaries.